website/src/components/detailsToggle/index.js

```diff
@@ -40,7 +40,7 @@ useEffect(() => {
         onMouseLeave={handleMouseLeave}
       >
         <span className={`${styles.toggle} ${isOn ? null : styles.toggleUpsideDown}`}></span>&nbsp;
-        <span>{alt_header}</span>
+        <span className={styles.headerText}>{alt_header}</span>
```

Comment:
adding span to separate alt_header and .disclaimer CSS and assign underline class to alt_header only.